### PR TITLE
TN11 parameter fine-tuning

### PR DESCRIPTION
**Pull request by michaelsutton**

### CODE REVIEW

File: protocol/flows/src/v5/blockrelay/flow.rs [hunk 1]
```diff
@@ -251,7 +251,10 @@ impl HandleRelayInvsFlow {
             .await?;
         let msg = dequeue_with_timeout!(self.msg_route, Payload::BlockLocator)?;
         let locator_hashes: Vec<Hash> = msg.try_into()?;
-        for h in locator_hashes {
+        // Locator hashes are sent from later to earlier, so it makes sense to query consensus in reverse. Technically
+        // with current syncer-side implementations (in both go-kaspa and this codebase) we could query only the last one,
+        // but we prefer not relying on such details for correctness
```
Comment (someone235) [inline protocol/flows/src/v5/blockrelay/flow.rs:256]: Should we add a TODO to send only one block in the next protocol version?

**Review by someone235 — CHANGES_REQUESTED** [review]

File: protocol/flows/src/v5/blockrelay/flow.rs (hunk 1 above)
Comment (michaelsutton) [reply protocol/flows/src/v5/blockrelay/flow.rs:256]: not sure you need a protocol version

File: consensus/core/src/blockstatus.rs [hunk 2]
```diff
@@ -43,4 +43,8 @@ impl BlockStatus {
     pub fn is_valid(self) -> bool {
         self != BlockStatus::StatusInvalid
     }
+
+    pub fn is_invalid(self) -> bool {
+        self == BlockStatus::StatusInvalid
```
Comment (coderofstuff) [inline consensus/core/src/blockstatus.rs:48]: Optional: seems strange that this isn't just `!is_valid()`

**Comment by coderofstuff** [comment]
Related to #354

File: consensus/core/src/blockstatus.rs (hunk 2 above)
Comment (michaelsutton) [reply consensus/core/src/blockstatus.rs:48]: I find it more readable

**Review by someone235 — APPROVED** [review]

**Pull request merged**